packages/terra-form-field/src/Field.jsx

```diff
@@ -61,6 +65,11 @@ const propTypes = {
    * Whether or not to append the 'optional' label to a non-required field label.
    */
   showOptional: PropTypes.bool,
+  /**
+   * Allows for custom inline styles in addition to (or separate from) the maxWidth style prop.
+   */
+  // eslint-disable-next-line react/forbid-prop-types
+  style: PropTypes.object,
```

Comment:
I'm not sure if we should expose style as a first class property, this will generate documentation for it within the example site props table of this component. Style can be read from props without exposing it as a prop-type